matador/workflows/castep/magres.py

```diff
@@ -150,6 +150,7 @@ def preprocess(self):
 
         # prepare to do pre-relax if there's no check file
         if os.path.isfile(self.seed + ".check"):
+            todo["scf"] = False
```

Comment:
I feel like this should still be turned on by default; the SCF here is meant to be "high quality" and uses a tighter tolerance than the relaxation. If the check file exists and the high quality SCF has already been run, then this should just load it in and complete.

Did you have a reason for disabling it?